packages/lab/src/__tests__/__e2e__/list-next/ListNext.cy.tsx

```diff
@@ -0,0 +1,317 @@
+    cy.findByText("list item 3").should("exist");
+  });
+
+  it("should allow a single item to be selected", () => {
+    cy.mount(<SingleSelectList />);
+
+    cy.findByRole("option", { name: ITEMS[1].label })
+      .click()
+      .should("have.attr", "aria-selected", "true");
+  });
+
+  it("should keep its selected item when the same item is selected", () => {
+    cy.mount(<SingleSelectList />);
+
+    cy.findByRole("option", { name: ITEMS[1].label })
+      .click()
+      .should("have.attr", "aria-selected", "true");
+    // click to select the same list item
+    cy.findByRole("option", { name: ITEMS[1].label })
+      .click()
+      .should("have.attr", "aria-selected", "true");
+  });
+
+  it("should deselect previous list item when a new list one is selected, for non deselectable", () => {
+    cy.mount(<SingleSelectList />);
+
+    cy.findByRole("option", { name: ITEMS[1].label })
+      .click()
+      .should("have.attr", "aria-selected", "true");
+    // click to select a different list item
+    cy.findByRole("option", { name: ITEMS[2].label })
+      .click()
+      .should("have.attr", "aria-selected", "true");
+    cy.findByRole("option", { name: ITEMS[1].label }).should(
+      "not.have.attr",
+      "aria-selected",
+      "true"
+    );
+  });
+
+  // Disabled list
+  describe("A disabled single select list", () => {
+    it("should not allow any items to be selected", () => {
+      cy.mount(<SingleSelectList disabled />);
+
+      cy.findAllByRole("option").should("have.attr", "aria-disabled", "true");
+    });
+
+    it("should not allow a disabled list item to be selected", () => {
+      cy.mount(
+        <SingleSelectList
+          children={
+            <>
+              <ListItemNext value={ITEMS[0].value}>
+                {ITEMS[0].label}
+              </ListItemNext>
+              <ListItemNext value={ITEMS[1].value} disabled>
+                {ITEMS[1].label}
+              </ListItemNext>
+              <ListItemNext value={ITEMS[2].value}>
+                {ITEMS[2].label}
+              </ListItemNext>
+            </>
+          }
+        />
+      );
+
+      cy.findByRole("option", { name: ITEMS[1].label }).should(
+        "have.attr",
+        "aria-disabled",
+        "true"
+      );
+    });
+
+    it("should not be focusable on mouse hover", () => {
+      cy.mount(
+        <SingleSelectList
+          children={
+            <>
+              <ListItemNext value={ITEMS[0].value}>
+                {ITEMS[0].label}
+              </ListItemNext>
+              <ListItemNext value={ITEMS[1].value} disabled>
+                {ITEMS[1].label}
+              </ListItemNext>
+              <ListItemNext value={ITEMS[2].value}>
+                {ITEMS[2].label}
+              </ListItemNext>
+            </>
+          }
+        />
+      );
+
+      cy.findByRole("option", { name: ITEMS[1].label })
+        .realHover()
+        .should("not.have.class", "saltListItemNext-focused");
```

Comment:
i'm a little confused. Do we expect that while hovering the item, it'll have the focused class applied?

Reply:
Ah good spot - that was one of the tests written in earlier stages, no longer applicable now will remove